doc/Type/DateTime.pod6

```diff
@@ -252,6 +252,51 @@ Returns an L<Instant|/type/Instant> object based on the invocant.
 
     say DateTime.new('2015-12-24T12:23:00+0200').Instant; # OUTPUT: «Instant:1450952616␤»
 
+=head2 method day-fraction
+
+Defined as:
+
+    method day-fraction(DateTime:D: --> Real:D)
+
+Returns the instant's time as a fraction of a 24-hour day.
+
+    say DateTime.new('2021-12-24T12:23:00.43Z').day-fraction; # OUTPUT: «0.5159772␤»
+
+Notice the C<day-fraction> value is the same as the fractional part of
+the C<modified-julian-date> for the same instant.
+
+=head2 method julian-date
+
+Defined as:
+
+    method julian-date(DateTime:D: --> Real:D)
+
+Returns the L<Julian Date|https://en.wikipedia.org/wiki/Julian_day> (JD) for the UTC date and time.
+
+    say DateTime.new('2021-12-24T12:23:00.43Z').julian-date; # OUTPUT: «2459573.0159772␤»
+
+The C<julian-date> starts at zero at the epoch starting at noon UTC on
+I<Monday, January 1, 4713 BC>, on the Gregorian calendar (the calendar
+in use in much of the world and in international commerce and travel)
+and is used in astronomy to define celestial objects' transits at the
+Earth's Prime Meridian.
+
+=head2 method modified-julian-date
+
+Defined as:
+
+    method modified-julian-date(DateTime:D: --> Real:D)
+
+Returns the L<Modified Julian Date|https://en.wikipedia.org/wiki/Julian_day> (MJD) for the UTC date and time.
+
+    say DateTime.new('2021-12-24T12:23:00.43Z').modified-julian-date; # OUTPUT: «59572.5159772␤»
+
+Notice the fractional part of the C<modified-julian-date> is same value as the C<day-fraction> for the same instant.
```

Comment:
*the* same value

Reply:
I was trying to emphasize the comaparison was within the same DateTime object and hence at the same instant.